### PR TITLE
Update budget window

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
OpenFisca-UK now only supports the 2022-27 budget window, dropping support for 2019-21. This updates all references and logic using the old budget window. Also ensures the OpenFisca-UK code runs locally (for me, soon hopefully for @jpycroft).

### CODE REVIEW

File: oguk/get_micro_data.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def get_calculator_output(baseline, year, reform=None, data=None):
 
     """
     # create a simulation
-    sim_kwargs = dict(dataset=dataset, year=2019)
+    sim_kwargs = dict(dataset=dataset, year=2022)
```
Comment (jdebacker) [inline oguk/get_micro_data.py:99]: Why doesn't this use the year argument passed into the function?  Without that, there will not be multiple years of data for the budget window (2022-2027).

File: oguk/get_micro_data.py [hunk 2]
```diff
@@ -106,7 +106,7 @@ def get_calculator_output(baseline, year, reform=None, data=None):
         print("Running current law policy baseline")
     else:
         print("Baseline policy is: ", reform)
-    
+
     sim.year = 2022
```
Comment (jdebacker) [inline oguk/get_micro_data.py]: Why isn't `sim.year` the same value as `year`?

File: oguk/get_micro_data.py (hunk 1 above)
Comment (nikhilwoodruff) [reply oguk/get_micro_data.py:99]: Here, the `year` argument specifies the named year of the dataset to use: OpenFisca-UK extrapolates future year results from earlier years. For example, the 2022 Enhanced FRS is generated from the 2022 FRS, SPI and LCFS datasets. If you call `sim.calc(variable, 2024)` it'll use the 2024 weights for the 2022 households and use 2024 policy.

File: oguk/get_micro_data.py (hunk 2 above)
Comment (nikhilwoodruff) [reply oguk/get_micro_data.py]: Good point: previously just to get something working. Removing.

Comment (nikhilwoodruff) [reply oguk/get_micro_data.py]: Although I think this was redundant anyway, as the `calc` calls all specify `period=year`.

**Comment by nikhilwoodruff** [comment]
@jdebacker just got the tests passing - OK to merge?

**Comment by rickecon** [comment]
Looks great to me. Thanks @nikhilwoodruff and @jdebacker.

**Comment by rickecon** [comment]
Looks great to me. Thanks @nikhilwoodruff and @jdebacker.

**Pull request merged**